processing/data_collection/gazette/spiders/rj_campos_goytacazes.py

```diff
@@ -0,0 +1,67 @@
+import re
+import dateparser
+
+from datetime import datetime
+from scrapy import Request
+from scrapy.exceptions import CloseSpider
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjCampoGoytacazesSpider(BaseGazetteSpider):
+    GAZETTE_ELEMENT_CSS = "ul.ul-licitacoes li"
+    TERRITORY_ID = "3301009"
+
+    allowed_domains = ["www.campos.rj.gov.br"]
+    name = "rj_campos_goytacazes"
+    start_urls = [
+        "https://www.campos.rj.gov.br/diario-oficial.php?PGpagina=1&PGporPagina=15"
+    ]
+    LIMIT_DATE = dateparser.parse("2015-01-01").date()
+
+    def parse(self, response):
+        """
+        @url https://www.campos.rj.gov.br/diario-oficial.php?PGpagina=1&PGporPagina=15
+        @returns requests 1
+        @returns items 15 15
+        @scrapes date file_urls is_extra_edition municipality_id power scraped_at
+        """
+
+        for element in response.css(self.GAZETTE_ELEMENT_CSS):
+            gazette_text = element.css("h4::text").extract_first() or ""
+
+            date_re = re.search(r"(\d{2} de (.*) de \d{4})", gazette_text)
+            if not date_re:
+                continue
+
+            # The extra edition for August 28th, 2018 has a typo in the month name.
+            date = date_re.group(0).replace("Agosoto", "Agosto")
+            date = dateparser.parse(date, languages=["pt"]).date()
+
+            if date < self.LIMIT_DATE:
```

Comment:
**question** Why you are limiting the date? If we have older gazettes, we should get it as well.

Reply:
Because [CONTRIBUTING.md](https://github.com/okfn-brasil/querido-diario/blob/main/CONTRIBUTING.md) states that the projec'ts goal is  to download gazettes since 2015:

> At this moment, the project aims for the 100 top Brazilian municipalities by population, collecting all the gazettes since 2015 

Campos Goytacazes City Hall's website provides gazettes up until 2010.

Should I remove this limit?

Reply:
Yes. Please, get all the files available. I think what that line was trying to say is that the spiders in the time of that writing was crawling files since 2015.

Reply:
@jvanz OK, I'll work on that!